Tests/BackendIntegrationTests/LoadShedderIntegrationTests.swift

```diff
@@ -37,15 +37,39 @@ class LoadShedderStoreKit1IntegrationTests: BaseStoreKitIntegrationTests {
         return Signing.enforcedVerificationMode()
     }
 
+    // MARK: -
+
+    private var logger: TestLogHandler!
+
+    override func setUp() async throws {
+        self.logger = TestLogHandler(capacity: 500)
+
+        try await super.setUp()
+    }
+
     func testCanGetOfferings() async throws {
         let receivedOfferings = try await Purchases.shared.offerings()
 
         expect(receivedOfferings.all).toNot(beEmpty())
         assertSnapshot(matching: receivedOfferings.response, as: .formattedJson)
     }
 
+    func testOfferingsComeFromLoadShedder() async throws {
+        self.logger.verifyMessageWasLogged(
```

Comment:
Having these kind of tests would be very useful in Android. It's difficult to test this without it